README.md

```diff
@@ -1,33 +1,147 @@
+# Astro Storyblok Starter Blueprint
 
-## Getting Started
+Integrate [Astro](https://astro.build/) and [Storyblok](http://www.storyblok.com) as a headless CMS.
 
-First, install all dependencies:
-```bash
+The blueprint is ideal for kick-starting new Storyblok and Astro projects. It features the following:
+- Pre-configured default blocks: `page`, `teaser`, `grid`, and `feature`.
+- Support for the Visual Editor's live preview.
+- Dynamic routing to fetch and render new stories automatically.
+- Rich text rendering and global content management.
+- Authentication-ready.
+- Internationalization-ready.
+- Minimal styling.
+
+> [!SUMMARY] Live preview
+> This feature provides real-time editing capabilities. [Learn how to enable the complete solution](https://github.com/storyblok/storyblok-astro?tab=readme-ov-file#enabling-live-preview-for-storybloks-visual-editor).
+
+## Get Started
+
+1. Clone this repository
+2. Open the project on your device
+3. Install dependencies
+
+```sh
 npm install
 ```
 
-Create an `.env` file with the following content (find your preview access token under **Settings** > **Access Tokens** in your Storyblok space):
-```bash
-NEXT_PUBLIC_STORYBLOK_TOKEN=your-preview-access-token
+### Authentication
+
+In the root of the project, create a `.env` file to store the Storyblok access token:
+
+```sh
+STORYBLOK_ACCESS_TOKEN="<REPLACE_WITH_YOUR_TOKEN>"
+```
+
+> [!INFO] API access token
+> Copy your space's preview access token from **Settings** > **Access Tokens**.
+> Learn more about Storyblok [access tokens](https://www.storyblok.com/docs/concepts/access-tokens).
+
+### Set the default environment
+
+To connect to the Storyblok Visual Editor, the local project must run over HTTPS.
+
+> [!INFO] HTTPS
+> Learn how to [create a local HTTPS certificate on macOS](https://www.storyblok.com/faq/setup-dev-server-https-proxy)
+
+To render a preview of the local project in the Visual Editor, follow these steps:
+
+1. Navigate to **Settings > Visual Editor**.
+2. Set the default environment to `https://localhost:3000`.
+3. Save.
+4. Open the `home` story.
+5. Click **Config**.
+6. Type `/` in the **Real path**.
+
+>[!TIP] Proxy command
+>Edit your `package.json` file to add a dedicated command:
+> > [!EXAMPLE]
+> > ```sh
+> > "proxy": "local-ssl-proxy --source 3000 --target 4321 --cert PATH/TO/localhost.pem --key PATH/TO/localhost-key.pem",
+> > "storyblok": "npm run dev & npm run proxy"
+> 
+> Change `PATH/TO/` to match the path of your `localhost.pem` and `localhost-key.pem` files.
+
+Run the development server with HTTPS enabled:
+```sh
+npm run storyblok
 ```
 
-Next, run the development server with HTTPS enabled:
-```bash
-npx next dev --experimental-https
+Back in Storyblok, open the **Home** story to start editing.
+
+Happy building!
+
+## Project Structure
```

Comment:
Isn't that implied already by the repository itself? Users can explore the structure directly.